packet_forwarder/inc/lora_pcap.h

```diff
@@ -0,0 +1,32 @@
+#ifndef _LORA_PKTFWD_LORA_PCAP_H
+#define _LORA_PKTFWD_LORA_PCAP_H
+
+#define __USE_POSIX199309 1 /* Have time.h emit struct timespec */ 
+/* fix an issue between POSIX and C99 */
+#if __STDC_VERSION__ >= 199901L
+    #define _XOPEN_SOURCE 600
+#else
+    #define _XOPEN_SOURCE 500
+#endif
```

Comment:
This pattern is something I've accepted in C source, but never understood. Or at least can't remember